mono_repo/test/travis_command_test.dart

```diff
@@ -580,17 +488,27 @@ os:
 - unneeded
 
 stages:
-- analyzer_and_format:
-  - group:
-    - dartfmt
-    - dartanalyzer: --fatal-warnings --fatal-infos .
-    dart: [dev]
-    os: [windows]
-  - group:
-    - dartfmt
-    - dartanalyzer: --fatal-warnings .
-    dart: [2.1.1]
-    os: [osx]
+  - analyze:
```

Comment:
I updated this test so the output of the shell script matches the frequently used `travisShellOutput` – so there's fewer things to update on any change